src/js/core/widget/core/Popup.js

```diff
@@ -630,8 +630,61 @@
 					// set state of popup
 					self.state = states.CLOSED;
 				}
+
+				if (self._ui.content.scrollHeight > self._ui.content.clientHeight) {
+					self._ui.footer.classList.add("bottomDivider");
+				}
 			};
 
+			prototype._onScroll = function (content) {
```

Comment:
You have twice the same method

Reply:
sorry. I will remove one function.

done.